resources/authgear/templates/en/web/authflowv2/settings_passkey.html

```diff
@@ -0,0 +1,103 @@
+{{ template "authflowv2/__settings_page_frame.html" . }}
+
+{{ define "page-navbar" }}
+  {{ template "authflowv2/__navbar.html"
+    (dict
+        "BackTitle" (translate "v2.component.navbar.default.item-back-button-label" nil)
+        "BackHref" (call $.MakeURL "/settings")
+        "Title" (translate "v2.page.settings-passkey.default.navbar-title" nil)
+    )
+  }}
+{{ end }}
+
+{{ define "page-content" }}
+<div
+  class="flex flex-col"
+  data-controller="authflow-passkey-creation"
+  data-authflow-passkey-creation-options-value="{{ $.CreationOptionsJSON }}"
+>
+  {{ $ctx := . }}
+  {{ range $.PasskeyIdentities }}
+    {{
+      template "authflowv2/__settings_action_item.html"
+      (dict
+        "Label" .Passkey.CreationOptions.PublicKey.User.DisplayName
+        "Description" ( include "description" . )
+        "ActionButton" (include "remove-btn" (dict "DialogID" .ID))
+      )
+    }}
+    {{ template "authflowv2/__settings_dialog.html" (dict
+      "Ctx" $ctx
+      "DialogID" .ID
+      "Title" (include "v2.page.settings-passkey.default.dialog-title" nil)
+      "Description" (include "v2.page.settings-passkey.default.dialog-description" nil)
+      "FormContent" (include "remove-input" (dict "ID" .ID "CSRFField" $.CSRFField))
+      "Buttons" (list
+        (dict
+          "Type" "Destructive"
+          "Label" (include "v2.component.button.default.label-remove" nil)
+          "Value" "remove"
+          "Event" "authgear.button.remove_passkey"
+        )
+        (dict
+          "Type" "Cancel"
+          "Label" (include "v2.component.button.default.label-cancel" nil)
+        )
+      )
+    )}}
+  {{ end }}
+  <button
+    type="button"
+    class="settings-link-btn py-6"
+    data-action="click->authflow-passkey-creation#create"
+    data-authflow-passkey-creation-target="button"
+    data-authgear-event="authgear.button.create_passkey"
+  >
+    {{ translate "v2.page.settings-passkey.default.add-passkey-button-label" nil }}
+  </button>
+  <form
+    class="hidden"
+    method="post"
+    novalidate
+    data-controller="turbo-form"
+    data-action="submit->turbo-form#submitForm"
+  >
+    {{ $.CSRFField }}
+    <input
+      type="hidden"
+      name="x_attestation_response"
+      data-authflow-passkey-creation-target="input"
+    />
+    <button
+      type="submit"
+      class="hidden"
+      name="x_action"
+      value="add"
+      data-authflow-passkey-creation-target="submit">
+    </button>
+  </form>
+</div>
+{{ end }}
+
+{{ define "description" }}
```

Comment:
We should have prefix to all internal template definition, e.g. "__settings_passkey_item_description". Otherwise, this template will override or will be overridden by others.